src/coinify/coinify.js

```diff
@@ -400,8 +427,17 @@ Coinify.prototype.request = function (method, endpoint, data) {
     options.headers['Authorization'] = 'Bearer ' + this._access_token;
   }
 
```

Comment:
I am not a fanboy of having all this network code here with repetition of encodedFormData, request etc... but I guess it is part of this idea of coinify being independent.